kong/clustering/wrpc_data_plane.lua

```diff
@@ -27,23 +26,25 @@ local DECLARATIVE_EMPTY_CONFIG_HASH = constants.DECLARATIVE_EMPTY_CONFIG_HASH
 local _M = {
   DPCP_CHANNEL_NAME = "DP-CP_config",
 }
+local _MT = { __index = _M, }
 
-function _M.new(parent)
+function _M.new(conf, cert, cert_key)
```

Comment:
👍 Big fan of replacing knowing the internals of the parent with just the params we need.